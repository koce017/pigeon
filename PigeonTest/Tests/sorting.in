10
-6
-1
1
-6
-8
3
-10
-13
7
8
---
15
23
-49
-56
5
49
22
-88
38
0
-37
5
-67
-5
18
-86