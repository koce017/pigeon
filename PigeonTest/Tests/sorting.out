-13
-10
-8
-6
-6
-1
1
3
7
8
---
-88
-86
-67
-56
-49
-37
-5
0
5
5
18
22
23
38
49